app/models/approval.rb

```diff
@@ -16,6 +16,7 @@ class Approval < ActiveRecord::Base
 
   scope :individual, -> { where(type: 'Approvals::Individual') }
 
+  validates :proposal, presence: true, allow_blank: true
```

Comment:
At this point in time, yep. Hypothetically, we might want to create some sort of "model" approval chain in the future, but that's a bit away.